client/src/extension.ts

```diff
@@ -169,9 +172,41 @@ async function ExecYseopCliCommandWithKbDirectoryPath(cliPath: string, ...words:
         window.showErrorMessage('This command must be used from within a workspace folder.');
         return;
     }
-
-    const workspaceFolders = workspace.workspaceFolders;
-    ExecYseopCliCommand(cliPath, ...words, workspaceFolders[0].uri.fsPath);
+    const workspaceFirstDir = workspace.workspaceFolders[0].uri.fsPath;
+
+    // Get the list of possible KBs in the workspace (like in library projects)
+    // using com.yseop.designer.prefs paths.
+    glob('**/.settings/com.yseop.designer.prefs', {
+        cwd: workspaceFirstDir,
+    })
+        .then((_matches) =>
+            // from “library/.settings/com.yseop.designer.prefs” generates “library”.
+            // from “.settings/com.yseop.designer.prefs” generates “.”
+            _matches.map((designerPrefsFile) => path.normalize(path.join(path.dirname(designerPrefsFile), '..'))),
```

Comment:
Why not dirname² instead of this weird mix of dirname+..?

Reply:
I just didn't think about it.
I'll try it and let you know.